js/WebView.android.js

```diff
@@ -131,6 +131,37 @@ class WebView extends React.Component<WebViewSharedProps, State> {
 
     let NativeWebView = nativeConfig.component || RNCWebView;
 
+    const compiledWhitelist = [
```

Comment:
I think this code does not belong here.
It is a duplicate of iOS code, so it should be shared inside WebViewShared.js as much a possible !

Reply:
I moved the code to WebViewShared. Hope it is still simple enough.